### PR TITLE
Remove web-dir arg

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

https://github.com/tektoncd/dashboard/issues/1564#issuecomment-654423196

Remove the web-dir arg since it's an implementation detail and
shouldn't need to be modified by users.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by AlanGreene** [comment]
@a-roberts @eddycharly what do you think? I've just hidden it for now and set the default value but think maybe we should remove it entirely.

**Comment by eddycharly** [comment]
@AlanGreene TBH, i would remove it entirely

**Comment by AlanGreene** [comment]
Yeah I think so too, will hold off for Adam's vote tomorrow then update the PR and get it in. Thanks

File: pkg/router/router.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ const ExtensionDisplayNameKey = "tekton-dashboard-display-name"
 // ExtensionRoot is the URL root when accessing extensions
 const ExtensionRoot = "/v1/extensions"
 
+var webResourcesDir = "/var/run/ko"
```
Comment (eddycharly) [inline pkg/router/router.go:58]: can be const

Comment (AlanGreene) [reply pkg/router/router.go]: updated the 2 below as well, thanks

**Comment by AlanGreene** [comment]
@a-roberts over to you. This should be a safe change, but let me know if I've missed anything.

**Comment by eddycharly** [comment]
@a-roberts /approve ?

**Comment by a-roberts** [comment]
But...

```
a-roberts approved these changes 16 hours ago
```

always works 😢 

/approve

**Comment by eddycharly** [comment]
Needs a rebase now.. :(

**Comment by a-roberts** [comment]
Ready to give this another nudge in once it's green 😄

**Pull request merged**